.gitignore

```diff
@@ -7,9 +7,12 @@ ondemand.sec
 virtualenv
 stage/*
 !stage/.keep
-docker-image/mock-cache.tar.gz
 .venv/
 packages/*/deb/build
 !packages/*/deb/build/.keep
 pkg/
-build/
+tmp/
+*.gem
+lib/ood_packaging/build_box/docker-image/Dockerfile
+lib/ood_packaging/build_box/docker-image/inituidgid.sh
+lib/ood_packaging/build_box/docker-image/install.sh
```

Comment:
Is this right?

Reply:
Yep, those 3 files are ERB templates and get rendered into the container build root so they can be used during building.  They are rendered here: https://github.com/OSC/ondemand-packaging/blob/5178b63b17020c832d17249c4ea26a49543838a3/lib/ood_packaging/build_box.rb#L85-L105